cellprofiler/modules/tile.py

```diff
@@ -261,6 +261,17 @@ def visible_settings(self):
             result += [self.add_button]
         return result
 
+    def help_settings(self):
+        result = [self.input_image, self.output_image, self.tile_method,
+                  self.wants_automatic_rows, self.rows,
+                  self.wants_automatic_columns, self.columns,
+                  self.place_first, self.tile_style,
+                  self.meander]
+
+        for additional in self.additional_images:
```

Comment:
We can omit this `for` loop. It will display text like:

```
Select an additional image to tile 
Select an additional image to tile? 
```

as many times as there are additional images.